tests/unit/components/playback/controls/AddCustomer.spec.js

```diff
@@ -120,9 +244,274 @@ describe("AddCustomer.enter_uuidbase57", () => {
       await Vue.nextTick();
       expect(spied_text_validator).toHaveBeenCalledWith(entry);
 
-      expect(target_error_message.text()).toStrictEqual(
-        spied_text_validator.mock.results[0].value
+      expect(target_error_message.text()).toStrictEqual(response_text);
+    }
+  );
+});
+
+describe("AddCustomer.enable_save_button", () => {
+  beforeEach(async () => {
+    const propsData = {
+      dialogdata: null,
+      dataindex: 0,
+    };
+    wrapper = mount(ComponentToTest, {
+      propsData,
+      store,
+      localVue,
+    });
+    store = await NuxtStore.createStore();
+  });
+
+  beforeAll(async () => {
+    // note the store will mutate across tests, so make sure to re-create it in beforeEach
+    const storePath = `${process.env.buildDir}/store.js`;
+    NuxtStore = await import(storePath);
+  });
+
+  afterEach(() => wrapper.destroy());
+
+  test.each([
+    [
+      "0VSckkBYH2An3dqHEyfRRE",
+      "The entered ID has an invalid character 0,",
+      "06ad547f-fe02-477b-9473-f7977e4d5e17",
+      "",
+      "Experiment anemia -1",
+      "",
+      "color: rgb(63, 63, 63); width: 250px; left: 250px;",
```

Comment:
the `width` and `left` attributes are not part of what you want to be testing here right?  Can you use `.contains` or some other assertion to just check that the color matches what you expect?

Reply:
https://vue-test-utils.vuejs.org/api/wrapper/#contains 

As per your earlier review never to use functions which have "Deprecation warning".
So as per earlier comment I haven't used .contains.

Reply:
Sorry, I should have clarified.  I meant the Jest assertion, not the Vue Test Utils wrapper.  https://jestjs.io/docs/en/expect#tocontainitem

`expect(cancel_btn.attributes().style).toContain("color: rgb(63, 63, 63);")`